### PR TITLE
docs: Review index and basics

### PR DESCRIPTION
**Pull request by dvsrepo**

### CODE REVIEW

File: docs/getting_started/basics.ipynb [hunk 1]
```diff
@@ -15,7 +15,7 @@
    "id": "14d13ee2-ffeb-46fa-9c62-d77c8328e499",
    "metadata": {},
    "source": [
-    "Here you will find some basic guidelines on how to get started with Rubrix."
+    "This guide will help you to get started with Rubrix to perform basic tasks such as uploading data, data annotation, and (soon) how to convert your data to train NLP models."
```
Comment (dvsrepo) [inline docs/getting_started/basics.ipynb]:
```suggestion
    "This guide will help you get started with Rubrix to perform basic tasks such as uploading data or data labelling
```

File: docs/getting_started/basics.ipynb [hunk 2]
```diff
@@ -894,15 +893,14 @@
    "source": [
     "![Defining a rule for a multi-label text classification task.](../_static/reference/webapp/define_rules_2.png)\n",
     "\n",
-    "Another approach to annotating your data is to develop heuristic rules tailored to your dataset. \n",
+    "Another approach to annotating your data is to define heuristic rules tailored to your dataset. \n",
     "For example, let us assume you want to classify news articles into the categories of *Finance*, *Sports*, and *Culture*. \n",
     "In this case, a reasonable rule would be to label all articles that include the word \"stock\" as *Finance*. \n",
     "\n",
-    "It is easy to see how you can quickly annotate vast amounts of data in this way, which we refer to as *weak supervision*. \n",
+    "It is easy to see how you can quickly annotate vast amounts of data in this way, s *weak supervision*. \n",
```
Comment (dvsrepo) [inline docs/getting_started/basics.ipynb]:
```suggestion
```

File: docs/getting_started/basics.ipynb (hunk 1 above)
Comment (dvsrepo) [inline docs/getting_started/basics.ipynb]:
```suggestion
    "This guide will help you to get started with Rubrix to perform basic tasks such as uploading data or data annotation.
```

File: docs/index.rst [hunk 3]
```diff
@@ -28,16 +27,14 @@ Key features:
...
-
 Quickstart
 ----------
 
 Getting started with Rubrix is easy, let's see a quick example using the 🤗 ``transformers`` and ``datasets`` libraries:
 
 .. code-block:: bash
 
-   pip install "rubrix[server]" "transformers[torch]" datasets
+   pip install "rubrix[server]==0.14.0" "transformers[torch]" datasets
```
Comment (frascuchon) [inline docs/index.rst]:
```suggestion
   pip install "rubrix[server]" "transformers[torch]" datasets
```

File: docs/index.rst [hunk 4]
```diff
@@ -179,20 +170,23 @@ You can join the conversation on our Github page and our Github forum.
...
+   tutorials/01-labeling-finetuning
+   tutorials/weak-supervision-with-rubrix
+   tutorials/02-spacy
+   tutorials/05-active_learning
+   tutorials/find_label_errors
+   tutorials/07-zeroshot_ner
+   tutorials/08-error_analysis_using_loss
+   tutorials/09-automatic_fastapi_log
+   tutorials/skweak
+   tutorials/weak-supervision-multi-label
```
Comment (frascuchon) [inline docs/index.rst]: Is this section updated? I have the impression that this structure is the old one.

Comment (dvsrepo) [reply docs/index.rst]: might be :-D I'll check this out

Comment (dvsrepo) [reply docs/index.rst]: Should be fixed

File: docs/index.rst [hunk 5]
```diff
@@ -4,20 +4,19 @@
    contain the root `toctree` directive.
 
 
-Welcome to Rubrix
-=================
+Rubrix documentation
+====================
 
-.. raw:: html
+`Rubrix <https://rubrix.ml>`_ is a **production-ready Python framework for exploring, annotating, and managing data** in NLP projects.
```
Comment (dvsrepo) [inline docs/index.rst]:
```suggestion
`Rubrix <https://rubrix.ml>`_ is a **production-ready framework for building and improving datasets** for NLP projects.
```

**Comment by dvsrepo** [comment]
https://rubrix.readthedocs.io/en/docs-house_keeping/

**Review by dcfidalgo — APPROVED** [review]

**Pull request merged**